### PR TITLE
CO-1635 Fixing pylint issues

### PR DESCRIPTION
**Pull request by vwvw**
One warning is left. The csv is used. Tried to disable the warning with a pylintrc without success.

### CODE REVIEW

File: hr_planning/models/resource_calendar.py [hunk 1]
```diff
@@ -12,27 +12,28 @@
...
+        res = super(ResourceCalendar, self).write(vals)
 
         if ('attendance_ids' in vals):
             self._generate()
         return res
 
-    @api.one
+    @api.multi
     def _generate(self):
+        self.ensure_one()
```
Comment (ecino) [inline hr_planning/models/resource_calendar.py]: You should better add a loop through the records in self. I see at line 31 that the method could be called on several records.

File: sponsorship_tracking/wizards/install_sds_tracking.py [hunk 2]
```diff
@@ -1,136 +1,138 @@
-# -*- coding: utf-8 -*-
```
Comment (ecino) [inline sponsorship_tracking/wizards/install_sds_tracking.py]: CLRF issue?

**Review by ecino — CHANGES_REQUESTED** [review]

File: sponsorship_tracking/wizards/install_sds_tracking.py (hunk 2 above)
Comment (vwvw) [reply sponsorship_tracking/wizards/install_sds_tracking.py]: Yup

**Comment by vwvw** [comment]
Should be fixed now

**Review by ecino — APPROVED** [review]

**Pull request merged**